doc/source/whatsnew/v0.9.0.rst

```diff
@@ -44,8 +39,10 @@ API changes
 
 .. ipython:: python
 
+   from pandas.compat import StringIO
```

Comment:
we do this as ``import io`` elsewhere, should be consistent @datapythonista

Reply:
agree